core/templates/pages/exploration-player-page/layout-directives/audio-bar.directive.html

```diff
@@ -252,6 +252,11 @@
     top: 3.5em;
   }
 
+  @media(max-width: 768px) {
```

Comment:
Have you tested other screen sizes?

Reply:
Yes done. (except ipad view for preview player as we cannot access it yet.)